test/CentOS-7-Linux-AppStream.repo

```diff
@@ -0,0 +1,31 @@
+[base]
+name=CentOS-7 - Base
+#mirrorlist=http://mirrorlist.centos.org/?release=7&arch=$basearch&repo=os&infra=$infra
```

Comment:
You can remove this since it’s in a comment no?

Reply:
That's a file copied from centos

Reply:
i think that you can't remove these lines and they are not treated as comments